README.md

```diff
@@ -1,8 +1,9 @@
 # Lino Core
 
-[![API Reference](https://godoc.org/github.com/cosmos/cosmos-sdk?status.svg
-)](https://docs.google.com/document/d/1Ytd57axPfJ13TSGVU_Yykv8ijW_VuWtx1s79ny6i5M8)
-[![LoC](https://tokei.rs/b1/github/lino-network/lino)](https://github.com/lino-network/lino).
+[![API Reference](https://godoc.org/github.com/cosmos/cosmos-sdk?status.svg)](https://docs.google.com/document/d/1Ytd57axPfJ13TSGVU_Yykv8ijW_VuWtx1s79ny6i5M8)
```

Comment:
Let's move these svg to our repo :)

Reply:
Got images from shields.io right now. I'm still figuring out how to store it in our customized url.

Reply:
For the go doc, we haven't have any go doc. I will generate our go doc.